### ISSUE TITLE
Train (pointnav) in an environment with added objects

### ISSUE THREAD

**Issue by ronjamoller** [facebookresearch/habitat-api#344]
## ❓ Questions and Help
HI, I have run the example trainings and would like to modify my environments by adding objects. I have gone through the baseline examples for the point goal navigation tasks and the object goal navigation tasks. Since i want to insert the object in relation to the goal position of the training yaml i cant just put it at a fixed position into the room. My understanding is that i can either
1. go from the habitat_baselines/run.py script with a modified version of a ppo yaml and modify the PPOTrainer .train() function as such that the objects are added after every environment reset - so i would "iterate" through the VectorEnv and modify all env. ... .sim objects accordingly. This is a bit complicated and I am not 100% certain I wont get in trouble with certain shared dependencies: Are all environments initialized only for one start/goal entry in the training yaml each or are they shared for optimization ?
2. go from the object goal task and modify those files, however I am unsure if my idea of "object goal" corresponds to the way the term is used in matterport - "objects" like fridge and Bed seem to correspond to certain parts of the room .glb and not actual separate object .glbs plus i would like to have full control over where my objects are inserted in relation to my goal gps coordinates.

tldr; can i manually add objects to environments during training without breaking something ? You have been incredibly helpful with all my questions so far and I hope someone has attempted a similar thing before.
Thank you,
Ronja Moeller
